repository-data/webfiles/src/main/resources/site/freemarker/common/catalogue/catalogue.ftl

```diff
@@ -102,7 +102,8 @@
                                     <span class="nhsd-m-selector-toggle-card__toggle">
                                         <div class="nhsd-a-selector-toggle">
                                         <@hst.link var="baseUrl"/>
-                                            <a href="<@renderUrl baseUrl=baseUrl retiredFilterEnabled=retiredFilterEnabled showRetired=!showRetired filters=filtersModel.selectedFiltersKeysMinusCollection(["retired-api","retired","retired-standard"]) />"
+                                            <a onclick="window.location = '<@renderUrl baseUrl=baseUrl retiredFilterEnabled=retiredFilterEnabled showRetired=!showRetired filters=filtersModel.selectedFiltersKeysMinusCollection(["retired-api","retired","retired-standard"]) />'"
```

Comment:
I'm not entirely sure why the anchor link with href was not working in Safari, but works ok now with the onClick as well.